### PR TITLE
Bump version from "1.7.2" to "1.8.0"

### PR DESCRIPTION
**Pull request by DilumAluthge**
cc: @quinnj 
cc: @mcmcgrath13

### CODE REVIEW

**Comment by quinnj** [comment]
I'll let @mcmcgrath13 chime in if we're ready for this release yet.

**Comment by mcmcgrath13** [comment]
In kicking the tires some more, I found a couple of issues addressed in #131.  Once that is merged in, I think we're good to release.

**Comment by DilumAluthge** [comment]
Now that https://github.com/quinnj/JSON3.jl/pull/131 is merged, is this good to be merged?

**Review by mcmcgrath13 — APPROVED** [review]

**Review by quinnj — APPROVED** [review]

**Pull request merged**
